### PR TITLE
hardhat doc improvements

### PR DESCRIPTION
**Pull request by Aniket-Engg**

### CODE REVIEW

File: docs/hardhat.md [hunk 1]
```diff
@@ -8,24 +8,40 @@ Remixd and Hardhat
 
 **Note:** If you have not used `remixd` before, read more about it [here](./remixd.html)
 
-If `remixd` is running locally on your device and shared folder is a **hardhat project**, an additional websocket plugin will be listening on port `65522`. According to its documentation,
+If `remixd` is running locally on your device and shared folder is a **Hardhat project**, an additional websocket plugin will be listening on port `65522`. According to its documentation,
 
 > _Hardhat projects are npm projects with the hardhat package installed and a hardhat.config.js file._
 
-The remixd hardhat listener is a websocket plugin similar to remixd and is used to perform Hardhat actions with Remix IDE. 
+So, it looks for `hardhat.config.js` file in shared folder, only then Hardhat websocket lister will run.
```
Comment (ryestew) [inline docs/hardhat.md:15]: Remixd looks for the `hardhat.config.js` file in shared folder, and if it finds it, the Hardhat websocket listener will run.

(listener was a not spelled correctly.  I wasn't so sure what "it" was referring to - did I get it correctly - when I replaced it with remixd?

Comment (ryestew) [reply docs/hardhat.md:15]: if not then just correct the mispelling.

**Review by ryestew — CHANGES_REQUESTED** [review]

File: docs/hardhat.md (hunk 1 above)
Comment (Aniket-Engg) [reply docs/hardhat.md:15]: Yes you got it right

Comment (Aniket-Engg) [reply docs/hardhat.md]: updated

**Review by ryestew — APPROVED** [review]

**Pull request merged**